content/en/docs/reference/using-api/api-concepts.md

```diff
@@ -753,15 +754,59 @@ rules:
 
 See [Authorization Overview](/docs/reference/access-authn-authz/authorization/).
 
-## Server Side Apply
+## Updating existing resources {#patch-and-apply}
+
+You can overwrite (**update**) an existing object using an HTTP PUT. It is
+the client's responsibility when sending a PUT to specify the `resourceVersion`,
+so that the API server can detect lost updates. In the event that the resource has
+changed (the `resourceVersion` the client provided is stale), the API server
+returns a 409 conflict response.
+
+As an alternative, the client can send an instruction to the API server to
+**patch** an existing resource, or to **apply** a set of changes.
+
+The Kubernetes API supports four different PATCH operations, determined by their
+corresponding HTTP `Content-Type` header:
+
+`application/apply-patch+yaml`
+: Server Side Apply YAML (a Kubernetes-specific extension, based on YAML).
+  All JSON documents are valid YAML, so you can also submit JSON using this
+  media type. See [Server Side Apply serialization](/docs/reference/using-api/server-side-apply/#serialization)
+  for more details.  
+  To Kubernetes, this is an **apply** operation.
+
+`application/json-patch+json`
+: JSON Patch, as defined in [RFC6902](https://tools.ietf.org/html/rfc6902).
+  A JSON patch is a sequence of operations that are executed on the resource;
+  for example `{"op": "add", "path": "/a/b/c", "value": [ "foo", "bar" ]}`.  
+  To Kubernetes, this is a **patch** operation.
+
+`application/merge-patch+json`
+: JSON Merge Patch, as defined in [RFC7386](https://tools.ietf.org/html/rfc7386).
+  A JSON Merge Patch is essentially a partial representation of the resource.
+  The submitted JSON is combined with the current resource to create a new one,
+  then the new one is saved.  
+  To Kubernetes, this is a **patch** operation.
+
+`application/strategic-merge-patch+json`
+: Strategic Merge Patch (a Kubernetes-specific extension based on JSON).
+  Strategic Merge Patch is a custom implementation of JSON Merge Patch.  
+  To Kubernetes, this is a **patch** operation.
+
+{{< note >}}
+The Kubernetes _server side apply_ mechanism has superseded Strategic Merge
+Patch.
+{{< /note >}}
 
 Kubernetes' [Server Side Apply](/docs/reference/using-api/server-side-apply/)
 feature allows the control plane to track managed fields for newly created objects.
 Server Side Apply provides a clear pattern for managing field conflicts,
-offers server-side `Apply` and `Update` operations, and replaces the
+offers server-side **apply** and **update** operations, and replaces the
 client-side functionality of `kubectl apply`.
 
-The API verb for Server-Side Apply is **apply**.
+The API verb for Server-Side Apply is **apply**, whereas the API verb for other
```

Comment:
I'm not exactly sure what this is supposed to mean?

Reply:
If I want to allow someone to make a change to an object using Server-Side Apply, what does the RBAC grant look like? Do I grant `apply`, or `patch`, or `update`?

Reply:
`patch`

Reply:
(if the object does not currently exist and server-side-apply would create the object, they need `create` permission as well)